src/timetrackings/timeTrackings.test.ts

```diff
@@ -74,8 +76,20 @@ describe('TimeTrackings', () => {
 
   test('start', async () => {
     mock.onPost(startPath).reply(200, { Success: true, NumResults: 1, Results: [{}] });
-    resultSingle = timeTrackings.start({ task_id: 1, user_id: 1 });
-    await resultSingle.then((result) => expect(result).toStrictEqual({}));
+    resultResource = timeTrackings.start({ task_id: 1, user_id: 1 });
+    await resultResource.then((result) =>
```

Comment:
We have an `await` here, so we don't need the `.then` and arrow function, let's just change it to:
```suggestion
    const result = await resultResource;
    expect(...)
```